### ISSUE TITLE
Unsupported call of .format()

### ISSUE THREAD

**Issue by guilyx** [gary30404/neural-network-from-scratch-python#1]
Hey,

In your Neural Network class, when you print your epochs, iterations and loss : 

`print('=== Epoch: {:d}/{:d}\tIteration:{:d}\tLoss: {:.2f} ===').format(epoch+1, self.num_epochs, iteration+1, loss)`

You call format on a NoneType i.e. print() instead of calling it on the string. It should be : 

`print('=== Epoch: {:d}/{:d}\tIteration:{:d}\tLoss: {:.2f} ==='.format(epoch+1, self.num_epochs, iteration+1, loss))`

Unless I am mistaken.

**Comment by chih-chun-chang** [comment]
Yes, you are correct.

**Comment by humerti** [comment]
A similar correction may need to be made to the last line (network.py):
`print('Test accuracy: {:.2f}%').format(acc*100) ` => wrong
`print('Test accuracy: {:.2f}%'.format(acc*100))`  => right
